src/assets/_project/_blocks/layout/head-assets-version-locked.html

```diff
@@ -10,7 +10,7 @@
 <link href="/assets/v4/{{version}}/css/qg-main.css" rel="stylesheet" type="text/css" media="all">
 
 <noscript>
-	<link href="/assets/v4/{{version}}/css/qg-noscript.css" rel="stylesheet" type="text/css" media="all">
+  <link href="/assets/v4/{{version}}/css/qg-noscript.css" rel="stylesheet" type="text/css" media="all">
 </noscript>
 
-<script src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
+<script src="https://code.jquery.com/jquery-3.6.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
```

Comment:
note that sha256 will deny this file since its now not matching.

Reply:
Thanks for the catch, should be fixed.